File: javascript/1101.js
var input = require('fs').readFileSync('/dev/stdin', 'utf8');
var lines = input.split('\n');

while (lines.length) {
    let sum = 0;
    let arrNum = [];
    var [x, y] = lines.shift().trim().split(' ').map((el) => parseInt(el));

    if (x > y) [x, y] = [y, x];
    if (x <= 0 || y <= 0) break;

    for (let i = x; i <= y; i++){
        arrNum.push(i);
        sum+=i;
    }
    console.log(`${arrNum.join(' ')} Sum=${sum}`);
}

File: python/1101.py
b=0
s=0
nums=''
soma=0
for x in range(1,999):
    nums=''
    soma=0
    lines=input().split(' ')
    if int(lines[0])<=0 or int(lines[1])<=0:
        break
    else:
        if lines[0]>lines[1]:
            b=int(lines[0])
            s=int(lines[1])
        else:
            b=int(lines[1])
            s=int(lines[0])
        for j in range(s,b+1):
            nums+=str(j)+' '
            soma+=j
    print(nums+'Sum='+str(soma))